src/providers/postgres/qgspostgresconn.cpp

```diff
@@ -1364,33 +1360,35 @@ int QgsPostgresConn::PQCancel()
 bool QgsPostgresConn::openCursor( const QString &cursorName, const QString &sql )
 {
   QMutexLocker locker( &mLock ); // to protect access to mOpenCursors
+  QString preStr = "";
```

Comment:
no need to initialize a blank string.
```suggestion
  QString preStr;
```